subprojects/packagefiles/mpdecimal/tests++/meson.build

```diff
@@ -0,0 +1,27 @@
+conf = configuration_data()
+
+if cc.has_header('pthread.h')
+  conf.set('HAVE_PTHREAD_H', 1)
+endif
+
+configure_file(
+  output: 'config.h',
+  configuration: conf)
+
+threads_dep = dependency('threads')
+
+runtest = executable('runtest', files('runtest.cc', 'test.cc'), dependencies: [mpdecpp_dep, threads_dep])
```

Comment:
`files()` does not need to be used here, because it's intended to be used in cases where you save the files to a variable and use them in another subdir. They then remember the directory they originally came from.

Here, they are inlined in the definition of executable() so there's no advantage of using it (and it makes the line length longer for no real advantage).

Reply:
It does not hurt either, does it? It is just 5 characters longer and I find it more explicit.

Reply:
I find it less readable, also, this line is already 105 characters and reducing it to 98 characters helps a *lot*.

Reply:
Whatever. It is nice that the review for wraps has been tightened so much after all the crap that entered the repo...

Reply:
It's a work in progress. :p

A few times I've just... gone through old wraps trying to systematically fix particular issues that bothered me. But there's a lot of old stuff. :/

Reply:
Unless you see any other issue with the PR, can you please merge this?